compiler/src/dotty/tools/dotc/core/TypeComparer.scala

```diff
@@ -297,7 +297,7 @@ class TypeComparer(initctx: Context) extends DotClass with ConstraintHandling {
         thirdTry
       case tp1: TypeParamRef =>
         def flagNothingBound = {
-          if (!frozenConstraint && tp2.isRef(defn.NothingClass) && state.isGlobalCommittable) {
+          if (!frozenConstraint && tp2.isRef(NothingClass) && state.isGlobalCommittable) {
```

Comment:
XXX This commit looks stale after the merge of kind-polymorphism, almost just a refactoring is left...